ui/app/mainui/ToastsManager.qml

```diff
@@ -241,7 +238,7 @@ QtObject {
             Global.openFinaliseOwnershipPopup(actionData)
             return
         case ToastsManager.ActionType.OpenSendModalPopup:
-            root.sendModalPopup.open()
+            Global.launchSendWithParams()
```

Comment:
No params here at all? :thinking:

Reply:
I dont know sadly.  I wasnt able to find a way to launch this at all, so kept it the way it was previously.